voltixApp/VoltixApp/Views/Keygen/PeerDiscoveryView.swift

```diff
@@ -188,6 +188,11 @@ struct PeerDiscoveryView: View {
             } label: {
                 PeerCell(id: peer, isSelected: viewModel.selections.contains(peer))
             }
+            .onAppear {
```

Comment:
The issue you linked , it was talking during keysign  , the change you are making is keygen

Reply:
I will apply it to both. It makes sense.